crypto/openssl/crypto/arm_arch.h

```diff
@@ -191,6 +191,10 @@ extern unsigned int OPENSSL_armv8_rsa_neonized;
 #    define PTR(n) x ## n
 #    define PTRN(n) n
 #   endif
+#  else
```

Comment:
`__ARM_ARCH__` will be 8 for arm32 with -march=armv8-a. Most assume armv8 == aarch64 but you can have A32+v8 too...

Reply:
Yeah, I had originally written the expression to assume that, but for FreeBSD at least these are built as armv7 still.  I guess I can restructure it a bit further.  It wasn't clear to me if __aarch64__ was portable across all compilers.